### PR TITLE
7.26c

### PR DESCRIPTION
**Pull request by POCKETMONEY99**
The whole patch. 

Some changes of note:
- Enigma Malefice Damage talent changed to +0.25 second stun duration (1.5 second stun max level in 2 seond intervals)
- Lion 15% Magic resistance changed to +15% mana drain slow. 
- Phantom Assassin 8% Coup de Grace Chance changed to 100% Coup de Grace Crit (fits better thematically but unsure if this is a buff or nerf to PA. I think it is a nerf). 
- Terroblade Reflection Cooldown Talent Removed. Support Terrorblade looks dead, its time for carry TB.
- Riki 25 talent changed from 70 agi to 125 smokescreen aoe.

### CODE REVIEW

**Comment by POCKETMONEY99** [comment]
2 other things I noticed while going through the files. CK level 10 30 attack speed talent is irrelevant on this hero. +10 all stats would be move suitable ( I havent altered this).
Was Venomancers +12 seconds plauge ward duration OP?, because +8% spell amp isnt great.

File: game/scripts/npc/abilities/clinkz_searing_arrows.txt [hunk 1]
```diff
@@ -39,7 +39,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage_bonus"                                    "30 40 50 60 80 160"
+        "damage_bonus"                                    "25 35 45 55 80 160"
```
Comment (lovesmuffins) [inline game/scripts/npc/abilities/clinkz_searing_arrows.txt:42]: shouldn't it be 75/150 at max level now.

File: game/scripts/npc/abilities/leshrac_pulse_nova.txt [hunk 2]
```diff
@@ -48,12 +48,6 @@
         "damage"                                          "100 180 260 500 740" //OAA
         "LinkedSpecialBonus"                              "special_bonus_unique_leshrac_3"
       }
-      "04" // unused
-      {
-        "var_type"                                        "FIELD_INTEGER"
-        "damage_scepter"                                  "160 190 220 450 680"
```
Comment (lovesmuffins) [inline game/scripts/npc/abilities/leshrac_pulse_nova.txt:54]: why u delete this

Comment (DarkoniusXNG) [reply game/scripts/npc/abilities/leshrac_pulse_nova.txt:54]: because Valve deleted it, it never worked anyway.

**Comment by DarkoniusXNG** [comment]
I added and edited some stuff:
* Axe lvl 15 talent: +2 Mana Regen replaced with +10% Spell Lifesteal
* Clinkz Searing Arrows damage reduced at OAA levels from 80/160 to 75/150.
* Clinkz Searing Arrows bonus damage talent increased from +15 to +20.
* Dark Willow lvl 10 talent: +10 INT replaced with +10% spell lifesteal.
* Nature's Prophet new lvl 10 talent doesn't work and its really bad. Replaced this talent with +20 Movement Speed. This is temporary ofc until we do a small rework of his ult.
* Ursa lvl 15 talent: +2 Mana Regen replaced with +20 Movement Speed
* Windranger lvl 10 talent: +1.5 Mana Regen replaced with +20 Damage.
* OD Essence Flux slow increased at OAA levels from 55%/65% to 60%/70%.
* OD Sanity Eclipse base damage increased at OAA levels from 625/900 to 650/950.

**Pull request merged**